src/www/build_cordova.action.sh

```diff
@@ -13,5 +13,34 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+PLATFORM=
+BUILD_MODE=
+for i in "$@"; do
+    case $i in
+    --platform=*)
+        PLATFORM="${i#*=}"
+        shift
+        ;;
+    --buildMode=*)
+        BUILD_MODE="${i#*=}"
+        shift
+        ;;
+    -* | --*)
+        echo "Unknown option: ${i}"
+        exit 1
+        ;;
+    *) ;;
+    esac
+done
 
-webpack --config=src/www/cordova.webpack.js ${BUILD_ENV:+--mode=${BUILD_ENV}}
+WEBPACK_MODE=
+case BUILD_MODE in
+    debug)
+        WEBPACK_MODE=development
+        ;;
+    release)
+        WEBPACK_MODE=production
+        ;;
+esac
+
+webpack --config=src/www/cordova.webpack.js ${WEBPACK_MODE:+--mode=${WEBPACK_MODE}}
```

Comment:
We could simplify by making the webpack script take our build mode. Or have it read BUILD_MODE. or have a script that converts the mode strings

Reply:
i'll do the script.